src/Bonsai.ML.Torch/IndexHelper.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using TorchSharp;
+
+namespace Bonsai.ML.Torch;
+
+/// <summary>
+/// Provides helper methods to parse tensor indexes.
```

Comment:
```suggestion
/// Provides helper methods to parse tensor indices.
```

This file seems to have a few occurrences of `indexes` throughout in case you want to amend for consistency.